.github/pull_request_template.md

```diff
@@ -7,6 +7,11 @@ For more information on the requirements for pull requests, please see [the CONT
 [] I confirm this contribution is my own and I agree to license it with Apache 2.0.
 [] I confirm the moderators may change the PR before merging it in.
 
+For new board enablements only.
+[] I confirm the board is Mbed enabled and passes the Mbed enabled test set.
```

Comment:
Please use capital letters for 'Mbed Enabled' and link to the requirements ;-)
https://www.mbed.com/en/about-mbed/mbed-enabled/introduction/

Reply:
Fixed.